Dockerfile

```diff
@@ -1,6 +1,6 @@
 # The runatlantis/atlantis-base is created by docker-base/Dockerfile.
 FROM runatlantis/atlantis-base:latest
-LABEL authors="Anubhav Mishra, Luke Kysow"
+LABEL authors="Anubhav Mishra, Luke Kysow, Jocelyn Thode"
```

Comment:
This is the authors of Atlantis, not the Dockerfile so please keep this as is.

Reply:
Done.